File: wwfpp/wwwroot/js/lang.js
/*******************************************************************************
* Package    : PandaPays
* Author     : eFour Core Pvt. Ltd.
* Author URI : https://www.efourcore.com.np/
* Page       : js/lang.js
* Description: maintain messages or labels
* Date       : 30-May-2026
* Update	 : 30-May-2026
*******************************************************************************
*******************************************************************************
* NOTE FOR TRANSLATING THIS FILE
*******************************************************************************
* DO NOT CHANGE THE TEXT OF LEFT HAND SIDE OF THE = (EQUAL) SIGN
* ONLY TRANSLATE THE TEXT OF RIGHT HAND SIDE OF = (EQUAL) SIGN within the "" (double quotation)
* DO NOT USE ' (SINGLE QUOTATION) OR " (DOUBLE QUOTATION) MARK
* DO NOT CHANGE THE TEXT WITH <[ ]>. EG : <[FISCAL-YEAR]> - <[FISCAL-YEAR]> NEED NOT TO BE CHANGED'
* DO NOT CHANGE THE TEXT WITH <>. EG : <H1> - H1 NEED NOT TO BE CHANGED'
*******************************************************************************/

/*----------------------------------------------------------------------------------------------------------*
 * MESSAGES FOR THE APPLICATION
 *----------------------------------------------------------------------------------------------------------*/
/* GLOBAL */
var please_wait = "Please wait ...";
var StrInitialMsg = "Alert !!! Check the following fields\n---------------------------------------------------";
var err_in_process = "Error happened during processing.";
var err_result_null = "Result is undefined/null";
var msg_preview = "Preview";
var msg_image_preview = "Image Preview";
var msg_close_preview = "Close Preview";
var msg_hours = "Hours";
//HOURS/MINUTE
var msg_lbl_hrs = "--Hour--";
var msg_lbl_min = "--Minute--";
var msg_enter_valid_email = "Enter valid email.";

var msg_clear = "Clear";
var msg_the_value = "The value ";
var msg_is_not_valid_date = " is not a vaild date.\n\r";
var msg_is_not_valid_date_format = " is not in a vaild date format.\n\r";
var msg_select_status = "Select status.";
var msg_select_sorting = "Select sorting.";
var msg_select_employee = "Select employee.";
var msg_select_fiscal_year = "Select fiscal year.";
var msg_select_start_fiscal_year = "Select start fiscal year.";
var msg_select_end_fiscal_year = "Select end fiscal year.";
var msg_select_year = "Select year.";
var msg_select_start_year = "Select start year.";
var msg_select_end_year = "Select end year.";
var msg_select_month = "Select month.";
var msg_select_start_month = "Select start month.";
var msg_select_end_month = "Select end month.";
var msg_select_week = "Select week.";
var msg_select_start_week = "Select start week.";
var msg_select_end_week = "Select end week.";
var msg_select_bi_week = "Select bi-week.";
var msg_select_start_bi_week = "Select start bi-week.";
var msg_select_end_bi_week = "Select end bi-week.";
var msg_select_empoloyee_payslip = "Please select at least one employee before previewing payslip.";

var msg_select_valid_csv_file = "Select valid CSV file.";
var msg_select_checkbox_to_activate = "Select checkbox to activate.";
var msg_select_checkbox_to_delete = "Select checkbox to delete.";
var msg_select_at_least_one_checkbox = "Select at least one checkbox to perform this action.";
var msg_select_at_least_one_checkbox_to_activate = "Select at least one checkbox to activate.";
var msg_select_at_least_one_checkbox_to_deactivate = "Select at least one checkbox to deactivate.";
var msg_select_at_least_one_checkbox_to_delete = "Select at least one checkbox to delete.";
var msg_select_at_least_one_employee_to_update = "Select at least one employee to update.";
var msg_select_at_least_one_record_to_update = "Select at least one record to update.";
var msg_select_showing_option = "Select showing option.";

var msg_enter_only_numeric_value = "Enter only numeric value.";
var msg_enter_only_alphabets_value = "Enter only alphabet value.";
var msg_enter_start_date = "Enter start date.";
var msg_enter_end_date = "Enter end date.";
var msg_please_enter_valid_date_format = "Enter a valid date in the format dd/mm/yyyy.";
var msg_please_enter_valid_date_format_mm = "Enter a valid date in the format mm/dd/yyyy.";
var msg_please_enter_date_in_the_format = "Enter date in the format dd/mm/yyyy.";
var msg_please_enter_date_in_the_format_mm = "Enter date in the format mm/dd/yyyy.";

var msg_select_at_least_one_checkbox_to_download = "Select at least one checkbox to download.";
var msg_select_at_least_one_checkbox_to_change_status = "Select at least one checkbox to change timesheet status.";
var msg_calculate_before_save = "Please click Calculate before Save.";
/* CONFIRMATION */
var msg_are_you_sure_to_delete = "Are you sure you want to delete ?";
var msg_are_you_sure_to_activate = "Are you sure you want to activate ?";
var msg_are_you_sure_to_deactivate = "Are you sure you want to deactivate ?";
var msg_are_you_sure_to_clear = "Are you sure you want to clear ?";
var msg_are_you_sure_to_perform_this_action = "Are you sure you want to perform this action ?\n\nClick OK button to perform this action or Cancel button to cancel process.";
var msg_are_you_sure_to_upload_selected_file = "Are you sure you want to upload selected file ?\n\nClick OK button to upload file or Cancel button to cancel process.";

/*----------------------------------------------------------------------------------------------------------*
 * USER ADMINISTRATION >>
 *----------------------------------------------------------------------------------------------------------*/
var username_policy = "Username must be at least 4 - 50 characters long, will accept A-Z, a-z, 0-9 and some punctuation(special) character( _, @, ., -). Please avoid space and/or unicode characters. ";
var password_policy = "Password must be at least 8 - 20 characters long. For Strong Password, it must have at least one upper case letter, one lower case letter,  one number and one punctuation(special) character(~ ` ! @ # $ % ^ & * ( ) - + = { } [ ] etc) . Please avoid space and/or unicode characters. ";
var pincode_policy = "Pin Code must be 6 digits random number. For strong pin code, avoid consecutive and/or repeating digits.";
var err_captcha_failed = "Fail, Captcha and entered text did not match.";
var msg_enter_username = "Enter username.";
var msg_blank_username = "No username.";
var msg_enter_password = "Enter password.";
var msg_enter_old_password = "Enter old password.";
var msg_enter_new_password = "Enter new password.";
var msg_enter_confirm_password = "Enter confirm password.";
var msg_enter_same_password = "Enter same password.";
var msg_invalid_username = "User name is not valid.";
var msg_select_user_level = "Select user level.";
var msg_old_new_pwd_same = "Old and New password are same. Please provide new one on new password.";
var msg_new_cpwd_not_same = "New password and confirm password are not same.";
var msg_new_pwd_used_in_past = "The new password provided has been used in the past. Please provide a password that is different from your last 5 passwords";
var msg_username_have_whitespace = "Username must not contain Whitespaces.";
var msg_username_have_unicode = "Username must not contain Unicode.";
var msg_username_have_invalid_chars = "Username contain disallowed Character.";
var msg_username_have_invalid_len = "Username must be 4-50 Characters Long.";
var msg_password_have_space = "Password must not contain Whitespaces.";
var msg_password_have_unicode = "Password must not contain Unicode.";
var msg_password_have_no_ucase = "Password must have at least one Uppercase Character.";
var msg_password_have_no_lcase = "Password must have at least one Lowercase Character.";
var msg_password_have_no_digit = "Password must have at least one Digit.";
var msg_password_have_no_spc = "Password must have at least one special Character.";
var msg_password_have_no_length = "Password must be 8-20 Characters Long.";
var msg_enter_captcha = "Enter CAPTCHA.";
var msg_enter_pin = "Enter Pin code.";
var msg_enter_old_pin = "Enter old Pin Code.";
var msg_enter_new_pin = "Enter new Pin Code.";
var msg_enter_confirm_pin = "Enter confirm Pin Code.";
var msg_enter_same_pin = "New Pin Code and Confirm Pin Code are not same.";
var msg_pin_code_should_have = "Pin Code should have 6 digits.";
var msg_pin_code_numeric_digits = "Pin Code should be numeric digits only.";
var msg_enter_login_step = "Login Step is required.";
var msg_one_step_msg = "You are Switching Login method to One Step Login method. It may be easy while Login but will have low Security which is not highly recommended.\n\n";
var msg_invalid_pin = "Invalid Pin Code";
var msg_old_new_pin_same = "Old and New Pin Codes are same. Please provide new one on new Pin Code.";
var msg_rslt_pwd_policy = "Weak Password";
var msg_rslt_pin_policy = "Weak Pin";
var msg_enter_same_pin = "New Pin Code and Confirm Pin Code are not same.";
var lbl_multi_login_emotp_sub = "A unique code will be sent to your registered email address for every login attempt.";
var lbl_multi_login_fixed_sub = "Define a personal PIN that you will use consistently for login.";
var msg_select_login_step = "Select Login Step.";
/**
 * Forgot password
 */
var err_inv_user_email = "Invalid Information.<br /><br /> Provide correct information and try again.";
var msg_suc_forgot_password = "Password Changed Successfully. You should soon receive an email with your login information. Please make sure to check your spam or junk folders if you can't find the email";
var err_forgot_pass_mail_fail = "Fail, there is problem sending email. Please contact with your server administrator.";
/**
 * Forgot PIN
 */
var msg_suc_forgot_pin = "Pin Code has been reset. You should soon receive an email with your login information. Please make sure to check your spam or junk folders if you can't find the email";
var err_inv_user_change_pin = "Invalid Email or Password or Old PIN Code."; 

var msg_data_exported_success = "Data exported Successfully.";
var msg_cant_edit_overtime_salary_processed = "Salary has already been processed for the selected period. You can't edit the overtime for this period.";

/*--------------------------------------------------------------------------------'
'* Payroll Administration > Dependent Allowance Distribution'
'*--------------------------------------------------------------------------------*/
var msg_dependent_allowance_cleared_scuccessfully = "Dependent allowance cleared successfully";
var msg_update_success = "Record(s) updated successfully."
var msg_some_error_occcured_please_try_again = "Some error has been occcured while processing the page! Please try again.";

/*--------------------------------------------------------------------------------'
'* Payroll Administration > LEAVE ACCRUAL'
'*--------------------------------------------------------------------------------*/
var msg_leave_accrual_cleared_scuccessfully = "Leave accrual cleared successfully";

/*--------------------------------------------------------------------------------'
'* Payroll Administration > GRATUITY ACCRUAL'
'*--------------------------------------------------------------------------------*/
var msg_gratuity_accrual_cleared_successfully = "Gratuity accrual cleared successfully";


var msg_date_required = "Date is required.";

File: wwfpp/Services/Lang.cs
//Access on Controller as " Lang.
public static class Lang
{
    public const string SITE_ADMIN_EMAIL  = ""; /* Site Administrator  email address*/
    public const string SITE_ADMIN_NAME   = "Administrator"; /* Site Administrator  name*/
    public const string SUPER_ADMIN_EMAIL = "shivascm@hotmail.com"; /* Super Administrator  email address*/
    public const string SUPER_ADMIN_NAME  = "Super Administrator"; /* Super Administrator*/

    public const string msg_not_able_connect_db = "Not able to connect to the database server.";

    //public const string lbl_fields_required = "* indicates required field";
    public const string msg_permission_denied = "Permission Denied!";
    public const string msg_please_wait = "Please wait ...";
    public const string msg_error = "Error in process.";
    public const string msg_error_invalid = "Invalid process.";
    public const string msg_undefined = "Status Undefined.";
    public const string msg_no_record_found = "No record found.";
    public const string msg_no_record_selected = "No record(s) selected.";
    public const string msg_insufficient_info = "Insufficient information.";
    public const string msg_session_expired = "Session expired. Please try again by re-login.";
    public const string msg_added_success = "Record(s) added successfully.";
    public const string msg_update_success = "Record(s) updated successfully.";
    public const string msg_activate_success = "Record(s) activated successfully.";
    public const string msg_deactivate_success = "Record(s) deactivated successfully.";
    public const string msg_delete_success = "[<DELETED-ROWS>] Record(s) deleted successfully.";
    public const string msg_deleted_some = "[<DELETED-ROWS>] record(s) deleted and [<UN-DEL-ROWS>] record(s) not deleted. The record(s) may be in use in another place. Please check and try again.";
    public const string msg_pending_deleted_some = "[<DELETED-ROWS>] pending record(s) deleted. [<UN-DEL-ROWS>] non pending record(s), cannot delete.";
    public const string msg_delete_fail = "(0) record(s) deleted. The record may be in use in another place. Please check and try again.";
    public const string msg_pending_delete_fail = "Non pending record(s), cannot delete.";
    public const string msg_record_exist_other = "Record(s) already exist in another record.";
    public const string msg_record_already_exist = "Request already exists for selected date range!";
    public const string msg_cancel_request_success = "Cancellation request has been submitted successfully.";
    public const string msg_cancel_request_fail = "Error while submitting cancellation request.";
    public const string msg_cancel_discared_success = "Cancellation request has been discarded successfully."; 
    public const string msg_cancel_discard_fail = "Error while dicarding cancellation request.";
    public const string msg_fs_used_greater_given = "Error while storing data.Used hours is greater than provided annual hours of employee.";
    public const string msg_leave_cf_success = "Leave carry forwarded successfully.";
    public const string msg_fy_mismatch = "Mismatched in fiscal year and/or date.";
    public const string msg_csv_import_success = "CSV file imported successfully.";
    public const string msg_email_sent_success = "Email Sent  successfully.";
    public const string msg_clear_success = "Record(s) cleared successfully.";
    public const string msg_leave_already_exists = "Leave already exists on the selected date range.";
    public const string msg_leave_hour_exceed = "Leave hour(s) exceeded. Make changes on the date range.";
    public const string msg_overtime_exist = "Overtime exist for the day. Select another date";
    public const string msg_overtime_not_enough_hour = "Not enough hours available for the day.";
    public const string msg_exceeded = "Maximum limit has been exceeded.";
    public const string msg_employee_overtime_bulk_error_save = "Nothing to save. All employees have zero overtime.";
    public const string msg_payslip_qued = "[<QUED-PAYSLIP-ROWS>] payslip(s) queued for sending.";
    public const string msg_no_payslip_found = "No payslips found to queue.";
    public const string msg_emp_leave_on_dayoff = "Leave cannot be applied on employee day-off.";
    public const string msg_emp_leave_on_holiday = "Leave cannot be applied on holiday dates.";


    public const string msg_invalid_user_pwd = $@"User Name or Password is invalid. Please try again.
                If you have forgotten the password, request a new password by clicking Forgot your password? link.";
    public const string msg_invalid_user_pin = $@"Pin is invalid. Please try again.
                If you have forgotten the pin, request a new pin by clicking Forgot your pin? link.";

    public const string msg_task_complete = "Task completed successfully.";
    public const string msg_some_fields_missing = "Some field(s) are missing. Complete all required inputs to continue.";
    public const string msg_error_generate_captcha = "Error Generating Captcha";
    public const string msg_incorrect_captcha  = "Captch is incorrect. Try again.";
    public const string msg_forgot_pwd_link_success = "Password reset link sent successfully.Please check your email inbox(and spam folder if needed) for the reset instructions.";
    public const string msg_forgot_pwd_link_invalid = "User Name or Email is invalid. Please try again.";
    public const string Username_policy = "Username must be at least 4 - 50 characters long, will accept A-Z, a-z, 0-9 and some punctuation(special) character( _, @, ., -). Please avoid space and/or unicode characters. ";
    public const string Password_policy = "Password must be at least 8 - 20 characters long. For Strong Password, it must have at least one upper case letter, one lower case letter,  one number and one punctuation(special) character(~ ` ! @ # $ % ^ & * ( ) - + = { } [ ] etc) . Please avoid space and/or unicode characters. ";
    public const string Pincode_policy = "Pin Code must be 6 digits random number. For strong pin code, avoid consecutive and/or repeating digits.";
    public const string lbl_empty_subject = "No Subject";
    public const string msg_new_cpwd_not_same = "New password and confirm password are not same.";
    public const string msg_pwd_length_not_valid = "Password must be between 8 and 20 characters long.";
    public const string msg_pwd_must_not_space = "Password must not contain spaces.";
    public const string msg_pwd_must_not_unicode = "Password must not contain Unicode characters.";
    public const string msg_pwd_must_upper = "Password must contain at least one uppercase letter.";
    public const string msg_pwd_must_lower = "Password must contain at least one lowercase letter.";
    public const string msg_pwd_must_digit = "Password must contain at least one digit.";
    public const string msg_pwd_must_special = "Password must contain at least one special character.";
    public const string msg_pwd_reset_expired = "Reset Password Link invalid or already expired.";
    public const string msg_pwd_reused_detected = "The new password you entered matches one of your previous passwords. For security, please choose a password different from your last five.";
    public const string msg_password_changed_successfully = "Password changed successfully!";
    public const string msg_suc_forgot_password = "Password changed successfully. Please proceed to the login page by clicking the Login button below.";
    public const string msg_pin_reset_expired = "Reset Pin Link invalid or already expired.";
    public const string msg_suc_forgot_pin = "Pin changed successfully. Please proceed to the login page by clicking the Login button below.";
    public const string msg_new_cpin_not_same = "New pin and confirm pin are not same.";
    public const string msg_rslt_pwd_policy = "Weak Password";
    public const string msg_rslt_pin_policy = "Weak Pin";
    public const string msg_old_new_pwd_same = "Old and new password are same. Please provide new one on new password.";
    public const string msg_invalid_password = "Password is invalid. Please try again with valid password.";
    public const string msg_pin_changed_successfully = "Pin changed successfully!";
    /*--------------------------------------------------------------------------------'
    * Change Password
    *--------------------------------------------------------------------------------'
    */

    public const string msg_password_expired = "The system found your password is Expired! So you are redirected here to change password.";
    public const string msg_password_weak = "The system found you are using WEAK password. So you are redirected here to change password.";
    public const string msg_next_password_change_due = "Your most recent password change was done on <[last-updated-date]> and upcoming password change is due in <[due-date]> day(s).";
    public const string msg_password_change_overdue = "Your most recent password change was done on <[last-updated-date]> and password change schedule exceeded by <[due-date]> day(s) from specified time limit of <[due-date-limit]> days, and it is now necessary to change your password.";
    public const string msg_no_password_change_info = "You have not any password changed record.";

    public const string lbl_single_login_step = "Single Factor Login";
    public const string lbl_multi_login_step = "Two Factor Login (Recomended)";
    public const string lbl_multi_login_emotp = "Receive OTP on email";
    public const string lbl_multi_login_emotp_sub = "A unique One Time PIN (OTP) will be sent to your registered email address for every login attempt.";
    public const string lbl_multi_login_fixed = "Set fixed PIN code";
    public const string lbl_multi_login_fixed_sub = "User Defined PIN (UDP) that you will use consistently for login.";
    public const string msg_MFA_changed_successfully = "Login step updated successfully.";
    /*--------------------------------------------------------------------------------'
    * Email Pin code for Second Step verification
    *--------------------------------------------------------------------------------'*/
    public const string lbl_no_holidays_defined = "Holidays not defined for this fiscal year";
    public const string msg_err_holiday_process = "Error in process. Selected date is past date or data already exist in timesheet.";
    public const string msg_provide_date_not_within_range = "Please provide the date within fiscal year <[FISCAL-START-DATE]> and <[FISCAL-END-DATE]>.";
    public static string msg_can_not_set_holiday_weekend = "Input date is weekend.";

    /*--------------------------------------------------------------------------------'
    * EMPLOYEE FUND SOURCE
    *--------------------------------------------------------------------------------'
    */
    public static string NO_FILE_UPLOADED = "No file uploaded.";
    public static string FISCAL_DATES_NOT_FOUND_IN_SESSION = "Invalid Start date and/or End date.";
    public static string FUND_SOURCE_NOT_EXIST = "Fund source does not exist: <[FUND-SOURCE-NAME]>.";
    public static string INACTIVE_EMPLOYEE = "The employee having code <[EMP-CODE]> is inactive or doesn't exist in our database.";
    public static string USED_HRS_GREATER_THAN_PROVIDED = "Used hours is greater than provided annual hours of employee <[EMP-CODE]>.";
    public static string EMPLOYEE_FUND_SOURCE_IMPORT_SUCCESSFUL = "Employee fund source import completed successfully.";
    public static string INVALID_ANNUAL_HOURS = "Fund source annual hours should be number.";

    /*
     * For Email
     */
    public const string EMAIL_ATTN_DO_NO_REPLY = "<br/><br/><b><i><font color=\"#B70000\" size=\"4\">ATTN : Please do not <u>reply</u> to this email. This is a system generated email sent by the system email address . This email address is not monitored and you will not receive any response.</font></i></b><br/><br/>";
    /*--------------------------------------------------------------------------------'
    * Email Pin code for Second Step verification
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_ACCOUNT_MULTI_STEP_PIN_SEND_SUBJECT = "<[SITE-TITLE]> - Multi-Factor Authentication code for <[ORG-NAME]>";
    public static string EMAIL_ACCOUNT_MULTI_STEP_PIN_SEND_MESSAGE = $@"
    Dear <[EMPLOYEE-NAME]>,<br>
    For your security, we have generated a multi-factor authentication code to verify your login.
    Please use the code below to complete your Login.<br><br>
    Your verification code: <[PIN-CODE]><br><br>
    Do not share this code with anyone.If you did not attempt to log in, please contact the IT helpdesk immediately.
    <br><br>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";
    /*--------------------------------------------------------------------------------'
    * Email Forgot Password link send
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_ACCOUNT_FORGOT_PWD_LINK_SUBJECT = "<[SITE-TITLE]> - Reset Password - <[ORG-NAME]>";
    public static string EMAIL_ACCOUNT_FORGOT_PWD_LINK_MESSAGE = $@"
    Dear <[EMPLOYEE-NAME]>,<br><br>
    You have requested to change your password.
    <br><br>
    To proceed, please click the secure link below:<br>
    <[CALL-BACK-URL]>
    <br><br>
    For your security, this link will expire in 30 minutes. If you did not request to change your password, you can safely ignore this email.
    <br><br>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";
    /*--------------------------------------------------------------------------------'
    * Email Notification about Password reset upon successful
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_ACCOUNT_PWD_RESETED_SUBJECT = "<[SITE-TITLE]> - Notification of Password Change - <[ORG-NAME]>";
    public static string EMAIL_ACCOUNT_PWD_RESETED_MESSAGE = $@"
    Dear <[EMPLOYEE-NAME]>,<br><br>
    This is to notify you that your account password has been successfully changed. If you made this change, no further action is required.<br>
    If you did not request this change, please reset your password immediately to secure your account.<br><br>
    For your security:<br>
    Do not reuse old passwords.<br>
    Choose a strong password with a mix of uppercase, lowercase, numbers, and special characters.<br><br>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";

    /*--------------------------------------------------------------------------------'
    * Email Forgot Pin link send
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_ACCOUNT_FORGOT_PIN_LINK_SUBJECT = "<[SITE-TITLE]> - Reset Pin - <[ORG-NAME]>";
    public static string EMAIL_ACCOUNT_FORGOT_PIN_LINK_MESSAGE = $@"
    Dear <[EMPLOYEE-NAME]>,<br><br>
    You have requested to change your Pin.
    <br><br>
    To proceed, please click the secure link below:<br>
    <[CALL-BACK-URL]>
    <br><br>
    For your security, this link will expire in 30 minutes. If you did not request to change your password, you can safely ignore this email.
    <br><br>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";
    /*--------------------------------------------------------------------------------'
    * Email Notification about Pin reset upon successful
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_ACCOUNT_PIN_RESETED_SUBJECT = "<[SITE-TITLE]> - Notification of Pin Change - <[ORG-NAME]>";
    public static string EMAIL_ACCOUNT_PIN_RESETED_MESSAGE = $@"
    Dear <[EMPLOYEE-NAME]>,<br><br>
    This is to notify you that your account pin has been successfully changed. If you made this change, no further action is required.<br>
    If you did not request this change, please reset your password and pin immediately to secure your account.<br><br>
    For your security:<br>
    Do not reuse old password/pin.<br>
    Choose a strong password with a mix of uppercase, lowercase, numbers, and special characters.<br>
    Choose a strong pin with a mix of digits. Avoid repeating the same digit or simple sequences.<br><br>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";
    /*--------------------------------------------------------------------------------'
    * Email Notification of Pay Slip
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_SALARY_PAY_SLIP_SEND_SUBJECT = "<[SITE-TITLE]> - Salary Pay Slip of period <[MONTH-NAME]>";
    public static string EMAIL_SALARY_PAY_SLIP_SEND_MESSAGE = $@"
    Dear <[EMPLOYEE-NAME]>,<br/><br/>
    This is to inform you that your salary for period <[MONTH-NAME]> has been processed.<br/><br/>
    Please login to <[SITE-TITLE]> to view the salary pay slip in detail.<br/><br/>
    <[TYPED-MESSAGE]><br/><br/>
    <[VIEW-LINK]><br/><br/>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";
    public static string EMAIL_SALARY_PAY_SLIP_SEND_MESSAGE_BLK = $@"
    Dear <[EMPLOYEE-NAME]>,<br/><br/>
    This is notification about your salary pay slip of period <[MONTH-NAME]><br/><br/>
    Your pay slip has been blocked due to some reason. Please contact Accounts Unit for more details.<br/><br/>
    <[TYPED-MESSAGE]><br/><br/>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";
    /*--------------------------------------------------------------------------------'
    * TIMESHEET- 	DUE NOTIFICATION TO FILL UP SOON '
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_EMPLOYEE_TIMESHEET_NTR_DUE_SUBJECT = "SITE_TITLE - Timesheet submission notification for period <[PEROID]>.";
    public static string EMAIL_EMPLOYEE_TIMESHEET_NTR_DUE_MESSAGE = $@"
    Respected Staff,<br/><br/>
    Please submit your timesheet for period <[PEROID]> as soon as possible.<br/><br/> 
    Please ignore this email if you have already submitted the timesheet for the same.<br/><br/>
    Thank You<br><[SITE-ADMIN-NAME]><br><[SITE-TITLE]> - <[ORG-NAME]>
    ";

    public const string msg_emp_manager_not_defined = "Your Manager OR/AND Line Manager has not been defined.";
    public const string msg_leave_apply_elligible = "Sorry, you are not elligible to apply for leave.";
    public const string msg_pending_leave_exist = "Some of your applied leaves are in pending. You can apply further leaves only after approving/declining pending leaves from your approval authority.";

    /*--------------------------------------------------------------------------------'
    * Email Notification about LEAVE DISCARD
    *--------------------------------------------------------------------------------'
    */
    public static string EMAIL_EMPLOYEE_CAN_DISCARD_SUBJECT = "Leave cancellation request discarded by <[EMPLOYEE-NAME-ONLY]>";
    public static string EMAIL_EMPLOYEE_CAN_DISCARD_MESSAGE = "Dear Sir/Madam,<br/><br/>Please discard my previously submitted following Leave cancellation request below.<br/><br/><[STR-MESSAGE]><br/><br/>Regards<br/><[EMPLOYEE-NAME-ONLY]><br/><br/>";

    /*--------------------------------------------------------------------------------'
    * Email Notification about TRAVEL CANCEL & DISCARD
    *--------------------------------------------------------------------------------'
    */
    public static string CANCEL_REQUEST_SAVED_AND_EMAIL_SENT = "Cancel request saved and email sent.";
    public static string EMAIL_EMPLOYEE_TRAVEL_CAN_SAVE_SUBJECT = "Travel cancellation request submitted by <[EMPLOYEE-NAME-ONLY]>";
    public static string EMAIL_EMPLOYEE_TRAVEL_CAN_SAVE_MESSAGE = "Dear Sir/Madam,<br/><br/>Please find my travel cancellation request below.<br/><br/><[STR-MESSAGE]><br/><br/>Regards<br/><[EMPLOYEE-NAME-ONLY]><br/><br/>";
    public static string NOT_FOUND = "Record not found.";
}